changelog/unreleased/kong/route-wildcard-snis.yml

```diff
@@ -0,0 +1,3 @@
+message: extend the field `snis` of the entity `routes` to support wildcard for the leftmost or the rightmost character. Note only the leftmost OR rightmost character can be the wildcard, otherwise the validation will fail.
```

Comment:
We should add a description that this feature is only available when kong runs with the `traditional_compatible` flavor.

Reply:
added